src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -149,10 +152,13 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
             const project = this.props.recentProjects.find((project) => project.id === projectId);
             await this.props.actions.loadProject(project);
             this.props.appTitleActions.setTitle(project.name);
+            await this.props.actions.updateProjectTagsFromFiles(this.props.project).then(() => {
+                this.setState({ tagsLoaded: true });
```

Comment:
only show tags once updated. Otherwise, any modifications to tags (delete, add, rename) will overwrite fields.json before they're loaded